routers/routes/routes.go

```diff
@@ -273,8 +273,6 @@ func RegisterRoutes(m *macaron.Macaron) {
 		}, openIDSignInEnabled)
 		m.Get("/sign_up", user.SignUp)
 		m.Post("/sign_up", bindIgnErr(auth.RegisterForm{}), user.SignUpPost)
-		m.Get("/reset_password", user.ResetPasswd)
```

Comment:
Do we need to atleast redirect to the new route before this is finally removed in 1.10?

Reply:
@adelowo I don't see why as emails having old route won't be usable anyway as code is valid only for few hours

Reply:
Ha true... stupid me